fitbenchmarking/templates/css/custom_style.css

```diff
@@ -1,5 +1,5 @@
 body {
-    background-color:#4D90BA;
+    background-color:#2e2f31;
```

Comment:
Is there a reason for this colour change?

Reply:
This was just a suggestion. I will leave it up to you to decide. I personally think that because the fitbenchmarking tables are very colourful, having a neutral background will be better. Having a grey background was making both the logo and the table pop more. It just focuses your attention on these elements. So that is why I updated this too.

Reply:
This makes sense to me, happy to change the background to grey :)